app/assets/stylesheets/functions/_modular-scale.scss

```diff
@@ -0,0 +1,116 @@
+@charset "UTF-8";
```

Comment:
I'm for removing `modular-scale` entirely. My only concern was of the "magic" nature of the numbers without this mixin.